catalog/transformer/catalog-transformer-common/src/main/java/ddf/catalog/transformer/common/tika/Mp4Attributes.java

```diff
@@ -0,0 +1,68 @@
+/**
+ * Copyright (c) Codice Foundation
+ * <p>
+ * This is free software: you can redistribute it and/or modify it under the terms of the GNU Lesser
+ * General Public License as published by the Free Software Foundation, either version 3 of the
+ * License, or any later version.
+ * <p>
+ * This program is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY; without
+ * even the implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU
+ * Lesser General Public License for more details. A copy of the GNU Lesser General Public License
+ * is distributed along with this program and can be found at
+ * <http://www.gnu.org/licenses/lgpl.html>.
+ */
+package ddf.catalog.transformer.common.tika;
+
+import java.util.Collections;
+import java.util.HashSet;
+import java.util.Set;
+
+import ddf.catalog.data.AttributeDescriptor;
+import ddf.catalog.data.MetacardType;
+import ddf.catalog.data.impl.AttributeDescriptorImpl;
+import ddf.catalog.data.impl.BasicTypes;
+
+/**
+ * This class provides attributes that reflect Mp4 attributes expected to be relevant to all metacard types
+ */
+public class Mp4Attributes implements MetacardType {
+    private static final Set<AttributeDescriptor> DESCRIPTORS;
+
+    private static final String NAME = "mp4";
+
+    private static final String EXT_PREFIX = "ext.mp4.";
+
+    public static final String AUDIO_SAMPLE_RATE = EXT_PREFIX + "audio-sample-rate";
+
+    static {
+        Set<AttributeDescriptor> descriptors = new HashSet<>();
+        descriptors.add(new AttributeDescriptorImpl(AUDIO_SAMPLE_RATE,
+                true /* indexed */,
+                true /* stored */,
+                false /* tokenized */,
+                false /* multivalued */,
+                BasicTypes.STRING_TYPE));
```

Comment:
Looks like it should be a number, right? https://tika.apache.org/1.8/api/org/apache/tika/metadata/XMPDM.html#AUDIO_SAMPLE_RATE